python/taichi/lang/ops.py

```diff
@@ -271,6 +271,19 @@ def _rsqrt(a):
     return _unary_operation(_ti_core.expr_rsqrt, _rsqrt, a)
 
 
+@unary
+def round(a):
+    """The round function.
+
+    Args:
+        a (Union[:class:`~taichi.lang.expr.Expr`, :class:`~taichi.lang.matrix.Matrix`]): A number or a matrix.
+
+    Returns:
+        Round of `a`.
```

Comment:
```suggestion
        The nearest integer of `a`.
```

Reply:
@strongoier Follow your advice, I modify the docstring for more precise meaning.